helloworld.py

```diff
@@ -0,0 +1 @@
+print('hello')
```

Comment:
Hey @tulucat
can you check this new line?

Reply:
yes